engine/src/main/java/com/datatorrent/stram/client/DTConfiguration.java

```diff
@@ -99,6 +102,11 @@ public ConfigException(String message)
     return result.entrySet().iterator();
   }
 
+  public Map<String, ValueEntry> getMap()
```

Comment:
DTConfiguration exposes iterators, that should be used. Map is an internal representation which should not be exposed.